packages/web-components/src/components/leadspace/leadspace.ts

```diff
@@ -197,21 +206,27 @@ class C4DLeadSpace extends StableSelectorMixin(LitElement) {
                       }
                     </linearGradient>
                   </defs>
-                  <rect class="${c4dPrefix}--leadspace__gradient__rect" width="100" height="100" />
+                  <rect class="${c4dPrefix}--leadspace__gradient__rect" part="gradient-rect" width="100" height="100" />
```

Comment:
I think we probably ought to leave the gradients alone here. Thoughts @m4olivei?

Reply:
It does seem a bit overkill. I can't see someone wanting to target parts of the SVG. There are plenty of other SVG icons in components where we wouldn't easily have the ability to add parts to the child elements of the SVG, only the SVG tag itself. My vote would be to just have a `part` attribute on the `<svg>` tag and end it there.